application/common/models/Method.php

```diff
@@ -177,4 +174,33 @@ public static function deleteExpiredUnverifiedMethods()
             }
         }
     }
+
+    /**
+     * @param integer $userId
+     * @param string $value
+     * @return Method
+     * @throws ConflictHttpException
+     * @throws ServerErrorHttpException
+     */
+    public static function create($userId, $value)
+    {
+        $method = Method::findOne(['value' => $value, 'user_id' => $userId]);
+
+        if ($method === null) {
+            $method = new Method;
```

Comment:
Very minor issue, but parentheses are recommended even when instantiating a class that doesn't take any arguments.
```suggestion
            $method = new Method();
```
I think the main place I learned this was the (pending) [PSR-12](https://github.com/php-fig/fig-standards/blob/master/proposed/extended-coding-style-guide.md#4-classes-properties-and-methods). Even though it hasn't been finalized, it seemed worth learning from it.